security/Security.php

```diff
@@ -326,6 +334,9 @@ public function init() {
 
 		// Prevent clickjacking, see https://developer.mozilla.org/en-US/docs/HTTP/X-Frame-Options
 		$this->getResponse()->addHeader('X-Frame-Options', $this->config()->frame_options);
+
+		// Prevent search engines from indexing the login page
+		$this->getResponse()->addHeader('X-Robots-Tag', $this->config()->robots_tag);
```

Comment:
Wrap this in an `if ($this->config()->robots_tag)` so devs can unset it without an empty header being sent